patches/chrome-browser-ui-BUILD.gn.patch

```diff
@@ -22,11 +22,19 @@ index 08ed584f529f787e4a19240fe7627441fd0f2d8a..519aa74da7169474bac95135a729573d
      }
    }
  
-@@ -5536,6 +5540,7 @@ static_library("ui_public_dependencies") {
+@@ -5345,6 +5349,7 @@ static_library("ui_public_dependencies") {
    if (enable_extensions_core) {
      public_deps += [ "//extensions/browser" ]
    }
 +  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
    if (!is_android) {
      public_deps += [
        "//build:branding_buildflags",
+@@ -5399,6 +5404,7 @@ source_set("layout_constants") {
+   if (is_chromeos) {
+     deps += [ "//chromeos/constants" ]
+   }
++  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_ui_layout_constants_public_deps allow_circular_includes_from = brave_chrome_browser_ui_layout_constants_allow_circular_includes_from
```

Comment:
there is definitely something wrong going on here, I need to update to 141 locally to take a look, but we should not be adding `allow_circular_includes_from` anywhere new